pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_cache.cc

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+#include "ai_djl_pytorch_jni_cache.h"
+
+jclass NULL_PTR_EXCEPTION_CLASS;
+jclass ILLEGAL_STATE_EXCEPTION_CLASS;
```

Comment:
Did we use this anywhere?

Reply:
just added the ILLEGAL_STATE_EXCEPTION_CLASS